cmd/gcp-controller-manager/node_syncer.go

```diff
@@ -35,25 +36,27 @@ import (
 
 const (
 	nodeSyncerControlLoopName = "node-syncer"
-	nodeSyncerQueueName       = "node-syncer-queue"
 	nodeSyncerResyncPeriod    = 30 * time.Minute
+	nodeSyncerGSARemoveDelay  = 30 * time.Minute
```

Comment:
Why is 30 minutes the right value here? What if a control plane is crash looping every 15 minutes? Would we expect service accounts to only be appended?

Reply:
First, I agree 30 mins is too long.  Let me change it to 10 mins which is still more than enough to handle the worst delay we have observed in test.

wrt controller crash looping, wouldn't it still get events based on when the delay should have expired?